src/viem/preparedTransactionSerialization.ts

```diff
@@ -0,0 +1,65 @@
+// Helper functions making prepared transactions serializable
+// so they can be used in redux actions (or even stores).
+import { TransactionRequestCIP42 } from 'node_modules/viem/_types/chains/celo/types'
+
+const bigIntProps = ['value', 'gas', 'maxFeePerGas', 'maxPriorityFeePerGas', 'gatewayFee'] as const
+
+type BigIntProps = (typeof bigIntProps)[number]
+
+type MapBigIntToString<T, K extends keyof T> = {
```

Comment:
nit: the naming's a little weird here because this type doesn't actually deal with bigint's, or even the keys we expect to have bigint values, it seems more general than that. Maybe it could be called MapSomePropsToString, something like that

Reply:
Right, I had started with checking bigint types and eventually changed it, I'll rename.